molecule/provisioner/ansible/playbooks/docker/create.yml

```diff
@@ -121,6 +121,8 @@
         hostname: "{{ item.hostname | default(item.name) }}"
         image: "{{ item.pre_build_image | default(false) | ternary('', 'molecule_local/') }}{{ item.image }}"
         pull: "{{ item.pull | default(omit) }}"
+        kernel_memory: "{{ item.kernel_memory | default(omit) }}"
```

Comment:
https://docs.docker.com/engine/reference/run/#kernel-memory-constraints#kernel-memory-constraints for reference. I've never had to touch kernel_memory, but having memory configurable is nice for some environments/tests.

Reply:
I only added `kernel_memory` as a "good to have"; i only really need `memory`. My use-case does not require `kernel_memory` so I will not have a chance to test it. Hence, removing it in this MR.

Some relevant docker warning:
`Docker warning: You specified a kernel memory limit on a kernel older than 4.0. Kernel memory limits are experimental on older kernels, it won't work as expected and
can cause your system to be unstable.`